generators/client/templates/angular/package.json.ejs

```diff
@@ -183,7 +183,7 @@
     "test:watch": "<%= clientPackageManager %> test <%= optionsForwarder %>--watch --clearCache",
     "webpack:dev": "<%= clientPackageManager %> run webpack-dev-server <%= optionsForwarder %>--config webpack/webpack.dev.js --inline --hot --port=9060 --watch-content-base --env.stats=minimal",
     "webpack:dev-verbose": "<%= clientPackageManager %> run webpack-dev-server <%= optionsForwarder %>--config webpack/webpack.dev.js --inline --hot --port=9060 --watch-content-base --profile --progress --env.stats=normal",
-    "webpack:build:main": "<%= clientPackageManager %> run webpack <%= optionsForwarder %>--config webpack/webpack.dev.js --env.stats=normal",
+    "webpack:build:main": "<%= clientPackageManager %> run webpack <%= optionsForwarder %>--config webpack/webpack.dev.js --env.stats=minimal",
```

Comment:
but this means you won't have the progress when running these outside of maven/gradle as well right? My goal was not to have the progress at all when running from maven gradle and have progress and stats when running via npm

Reply:
ok just tried this and indeed it looks good on local console so I think I like this better, i'll close my PR

Reply:
So it seems like we agree there. So let's merge !